python/tissue_analysis.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
If you don't need this script anymore, you can delete it. If you want to keep it, you can edit your .gitignore file in the top of the egem folder and type in the path to the file. That will cause git to not add this file when pushing.

Reply:
Going to delete soon just at the end to be sure